### PR TITLE
Initial cypress.io setup. Login test. Implement run script.

### PR DESCRIPTION
**Pull request by timestopper**

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
```
Comment (christinecha) [inline cypress/plugins/index.js:1]: Can we get rid of all these placeholder comments?

File: cypress/integration/login_spec.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  
+      // return false to prevent the error from
+      // failing this test
+      return false
+    })
+
+    cy.get('button[type="submit"]').click();
+  })
+
+  it('Login attempt success', ()=>{
```
Comment (christinecha) [inline cypress/integration/login_spec.js:31]: Is your eslint set up? I don't have a pre-commit hook or anything so I just set my editor to lint on save.

File: package.json [hunk 3]
```diff
@@ -21,7 +21,9 @@
     "start": "NODE_ENV=production node ./server/index.js",
     "deploy:staging": "NODE_ENV=staging yarn run set:heroku-config && ./scripts/deploy-staging.sh",
     "deploy:production": "NODE_ENV=production yarn run set:heroku-config && ./scripts/deploy-production.sh",
-    "deploy:all": "yarn run deploy:staging & yarn run deploy:production"
+    "deploy:all": "yarn run deploy:staging & yarn run deploy:production",
+    "cy:run": "cypress run",
```
Comment (christinecha) [inline package.json:25]: let's change this command to just "test" and remove the one below - I think ideally we would have a script `test` where we can run it with flags, like `yarn run test --file=cypress/integration/login_spec.js`

File: cypress/fixtures/example.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (christinecha) [inline cypress/fixtures/example.json:1]: I don't think we are using this file - we can remove

**Comment by timestopper** [comment]
Fixed based on comments.

**Comment by timestopper** [comment]
This changes already included in PR6:
https://github.com/pingpod/pingpod-web/pull/6
We can close this PR.

**Pull request closed without merging**